src/reader/mod.rs

```diff
@@ -1573,16 +1571,6 @@ mod test {
                 use crate::reader::Reader;
                 use pretty_assertions::assert_eq;
 
-                #[$test]
-                $($async)? fn start_text() {
-                    let mut reader = Reader::from_str("bom");
-
-                    assert_eq!(
-                        reader.$read_event($buf) $(.$await)? .unwrap(),
-                        Event::StartText(BytesText::from_escaped("bom").into())
-                    );
-                }
-
```

Comment:
Need to replace the `text` test content with the body of this test (with replacing of `StartText` to `Text` and `"bom"` to `"text"` of course).

Also, it would be worth to add a test for a text with only BOM:
```rust
let mut reader = Reader::from_str("\u{feff}");
assert_eq!(
    reader.$read_event($buf) $(.$await)? .unwrap(),
    // what to expect?
);
```